.github/workflows/CI.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     needs: [style]
     strategy:
       matrix:
-        rust: [ 1.63 ] # keep in sync with 'rust-version' in Cargo.toml
+        rust: [ 1.64 ] # keep in sync with 'rust-version' in Cargo.toml
```

Comment:
It's possible to keep MSRV the same way I did in the accompanying hyper PR (re-implement `poll_fn`, import `ready!` from `futures_core`) if that's preferred.

Reply:
Just checked, Debian stable is still on 1.63. While I don't explicitly document that we track Debian, it is one of the gauges I use to determine if it's worth increasing MSRV.

Reply:
Rebased and updated to keep MSRV as-is.